frontend/src/entities/dataset/dataset.columns.tsx

```diff
@@ -47,58 +43,8 @@ const defaultColumns: TableProps.ColumnDefinition<IDataset>[] = [
     },
 ];
 
-const defaultFileColumns: TableProps.ColumnDefinition<IDatasetFile>[] = [
```

Comment:
These were specific columns for displaying files. Now that files are listed in the DatasetBrowser we have built in defaults and allow optional additional columns. This file still exists because it has `defaultColumns` used for displaying the listing page for Datasets themselves.